packages/create-graphql/src/utils.js

```diff
@@ -40,3 +43,7 @@ export const verifyYeoman = async () => { // eslint-disable-line import/prefer-d
 
   return true;
 };
+
+export const verifyVersion = async () => { // eslint-disable-line import/prefer-default-export
```

Comment:
There's no need for the `eslint-disable-line` comment, it will no longer throw an error on this file since there are two exports, it's better to remove both of the comments.

Reply:
Also, since it is a one-line function, use a shorthand removing the async/await, brackets & return syntaxes.